### PR TITLE
Further update of Praha IPR aerial imagery

### PR DESCRIPTION
**Pull request by Vlada-C**
Applying similar corrections and improvements as in the recently edited "IPR Praha low-vegetation" file to the corresponding "IPR Praha latest" file. Copying the more detailed bounding area geometry from the above mentioned file (not sure whether it is 100% correct but seems more accurate than the simple box).

### CODE REVIEW

File: sources/europe/cz/IPR-orotofoto-vege-tms.geojson [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
+        "description": "Yearly IPR Praha orthophotos, taken after the leaves are fallen from the trees. Same imagery as Praha IPR low-vegetation orthophoto but served from a TMS proxy",
+        "type": "tms",
         "url": "https://osm-{switch:a,b,c}.zby.cz/tiles_ipr_vege.php/{zoom}/{x}/{y}.jpg",
+        "country_code": "CZ",
+        "permission_osm": "explicit",
+        "license_url": "https://wiki.openstreetmap.org/wiki/Cs:%C4%8Cesko/freemap#Mapov%C3%A9_podklady_IPR_Praha",
+        "privacy_policy_url": "https://iprpraha.cz/stranka/95/informace-o-zpracovani-osobnich-udaju",
+        "attribution": {
+            "url": "https://iprpraha.cz/",
+            "text": "IPR Praha"
```
Comment (matheusgomesms) [inline sources/europe/cz/IPR-orotofoto-vege-tms.geojson]:
```suggestion
            "text": "IPR Praha; OSM CZ"
```

File: sources/europe/cz/IPR-orotofoto-last-tms.geojson [hunk 2]
```diff
@@ -1,40 +1,186 @@
...
-        "min_zoom": 1,
-        "license_url": "https://wiki.openstreetmap.org/wiki/WikiProject_Czech_Republic/freemap",
         "country_code": "CZ",
-        "type": "tms"
+        "permission_osm": "explicit",
+        "license_url": "https://wiki.openstreetmap.org/wiki/Cs:%C4%8Cesko/freemap#Mapov%C3%A9_podklady_IPR_Praha",
+        "privacy_policy_url": "https://iprpraha.cz/stranka/95/informace-o-zpracovani-osobnich-udaju",
+        "attribution": {
+            "url": "https://iprpraha.cz/",
+            "text": "IPR Praha"
```
Comment (matheusgomesms) [inline sources/europe/cz/IPR-orotofoto-last-tms.geojson]:
```suggestion
            "text": "IPR Praha; OSM CZ"
```

**Review by matheusgomesms — COMMENTED** [review]
Thanks for your work!

Shall we update the attribution text, only for the mirrors, adding OSM CZ community? Since you guys are hosting them, I think it's valid to also attribute the community (we did that in Brazil, for example).

If you agree, just please accept the changes and I'll merge the PR. The rest looks good to me!

**Comment by Vlada-C** [comment]
> Thanks for your work!
> 
> Shall we update the attribution text, only for the mirrors, adding OSM CZ community? Since you guys are hosting them, I think it's valid to also attribute the community (we did that in Brazil, for example).
> 
> If you agree, just please accept the changes and I'll merge the PR. The rest looks good to me!

Thanks again, Matheus, for reviewing the changes. I believe we are finished with IPR for now.

**Pull request merged**